### ISSUE TITLE
Removing depracated services

### ISSUE THREAD

**Issue by Zodiac1978** [ArthurGareginyan/all-meta-tags#6]
Do you have any plans to remove the services mentioned on: https://wordpress.org/support/topic/some-obsolete-services/ ?

Google+ is only for Enterprise clients available and not for public use anymore.
https://en.wikipedia.org/wiki/Google%2B or https://plus.google.com/

Alexa has retired the service 2016:
https://support.alexa.com/hc/en-us/articles/219135887-Claiming-has-been-retired-May-2016

Web of Trust 
The link is not working for me: https://www.mywot.com/ (not a 404, but completely gone)

Additionally https://www.specificfeeds.com/ is redirecting to https://follow.it/intro and the new service seems to not need a verification.

And the keywords are completely ignored from Google since 2009:
https://developers.google.com/search/blog/2009/09/google-does-not-use-keywords-meta-tag

Happy to help with PRs if those services could be removed.

**Comment by ArthurGareginyan** [comment]
Yes, we decided to remove these services. PRs would be very much appreciated. Thank you in advance!
